### PR TITLE
ElasticSearch Rest  BulkIndex NPE fix for Race condition

### PR DESCRIPTION
**Pull request by mdepak**
As per https://github.com/Netflix/conductor/issues/1553, there is a race condition in `BulkRequestBuilder` class of ElasticSearch which is used in ElasticSearchDAO transport client. In our setup, this NullPointerException happens in case of ElasticSearchRestDAO as well. ElasticSearchRestDAO uses `BulkRequest` object which in turn uses `ArrayList` to maintain the index requests is not synchronized. Hence this problem occurs in Rest implementation as well. So using an approach similar to https://github.com/Netflix/conductor/pull/1554 to use thread-safe wrapper for Rest implementation as well. Verified that in high throughput scenario the NullPointerExceptions are happening no more. Thanks, @skissane for the initial solution.

### CODE REVIEW

**Comment by apanicker-nflx** [comment]
@skissane Referencing your [comment](https://github.com/Netflix/conductor/pull/1554#issuecomment-596289037) from #1554 , can you please comment on this pull request.

File: es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/BulkRequestWrapper.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.netflix.conductor.dao.es5.index;
```
Comment (apanicker-nflx) [inline es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/BulkRequestWrapper.java:1]: Please add a license header.

File: es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/BulkRequestWrapper.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package com.netflix.conductor.dao.es6.index;
```
Comment (apanicker-nflx) [inline es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/BulkRequestWrapper.java:1]: Please add a license header.

File: es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/BulkRequestWrapper.java (hunk 1 above)
Comment (mdepak) [reply es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/BulkRequestWrapper.java:16]: Added license header.

File: es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/BulkRequestWrapper.java (hunk 2 above)
Comment (mdepak) [reply es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/BulkRequestWrapper.java:16]: Added license header.

**Pull request merged**